package.json

```diff
@@ -1604,12 +1604,6 @@
                     "description": "When opening ipynb files, automatically preview the contents in the Interactive window.",
                     "scope": "resource"
                 },
-                "jupyter.useNotebookEditor": {
```

Comment:
Do we need to mark this as deprecated somehow? Is there a way to signal to people that this setting is no longer supported if they have in their settings.json?

Reply:
Yeah, our runMagicCommands has an example of this. I believe that we want to use that instead of removing it fully.

Reply:
Do'nt see the point of marking it as deprecated, what's the user going to do. Its not like they can go back to an older behavior.